lib/decimal.ex

```diff
@@ -267,8 +267,16 @@ defmodule Decimal do
     num2
   end
 
+  def add(%Decimal{coef: :inf, sign: sign, exp: exp1} = num1, %Decimal{coef: :inf, sign: sign, exp: exp2}) do
+    if exp1 > exp2 do
+      num1
+    else
+      num2
```

Comment:
Ah, missed it in some iteration. I've added it and amended it to the proper commit.